source/postprocess/topography.cc

```diff
@@ -87,9 +92,12 @@ namespace aspect
                 if ( cell->face(face_no)->boundary_id() != relevant_boundary)
                   continue;
 
-                for ( unsigned int v = 0; v < GeometryInfo<dim>::vertices_per_face;  ++v)
+                face_vals.reinit( cell, face_no);
+
+                for (unsigned int corner = 0; corner < face_corners.size(); ++corner)
                   {
-                    Point<dim> vertex = cell->face(face_no)->vertex(v);
+                    Point<dim> vertex = face_vals.quadrature_point(corner);
```

Comment:
Oh, good spot. I would not have thought of looking for these kinds of places as well!